js/script.js

```diff
@@ -439,6 +439,7 @@ let secret, currentQuestion, charactersInPlay
 
```

Comment:
I think if you would write let currentQuestion = { category: 'hair', value: 'brown' } here you would be able to choose that also on the first guess, which you cant as it is now.